lit_llama/utils.py

```diff
@@ -327,13 +327,11 @@ def __init__(self, obj, saver, protocol_version=5):
         if isinstance(obj, torch.storage.TypedStorage):
             # PT upstream wants to deprecate this eventually...
             storage = obj._untyped_storage
-            storage_dtype = obj.dtype
             storage_type_str = obj._pickle_storage_type()
             storage_type = getattr(torch, storage_type_str)
             storage_numel = obj._size()
         else:
             storage = obj
-            storage_dtype = torch.uint8
             storage_type = normalize_storage_type(type(obj))
```

Comment:
The import for this was missing, so clearly this code path is not used.

Reply:
Yes, and removing the unused `storage_dtype` is great but I would expect a future update of Pytorch to hit the second part. (but `normalize_storage_type` needs to be qualified with `torch.serialization`.... .ehem.)